app/app.go

```diff
@@ -106,7 +111,9 @@ var (
 	_           servertypes.Application = (*AkashApp)(nil)
 
 	// module accounts that are allowed to receive tokens
-	allowedReceivingModAcc = map[string]bool{}
+	allowedReceivingModAcc = map[string]bool{
+		escrowtypes.ModuleName: true, // depositors transfer tokens to module through authz
+	}
```

Comment:
This needed to be enabled in order for escrow module to receive funds from depositor through the authz module.